crates/trigger/src/runtime_config.rs

```diff
@@ -515,3 +542,41 @@ mod tests {
         }
     }
 }
+
+// parsed client tls options
+#[derive(Debug, Clone)]
+pub struct ParsedClientTlsOpts {
+    pub components: Vec<String>,
+    pub hosts: Vec<String>,
```

Comment:
Tests and documentation for what happens when multiple TLS blocks contain overlapping hosts would be pretty useful

Reply:
Technically we should allow multiple certs to be configured; the tls negotiation can select from multiple.

Reply:
in that case maybe we keep it simple and let the last block win (the current behavior) or should we fail it for now?

Reply:
I might suggest first-wins here since that is how variable provider configs already work.

Reply:
👍 , I have now changed the logic to make first entry win.